pkg/cmd/pipelinerun/delete.go

```diff
@@ -78,22 +77,8 @@ func deletePipelineRuns(s *cli.Stream, p cli.Params, prNames []string) error {
 	if err != nil {
 		return fmt.Errorf("failed to create tekton client")
 	}
-
-	var errs []error
-	var success []string
-	for _, prName := range prNames {
-		if err := cs.Tekton.TektonV1alpha1().PipelineRuns(p.Namespace()).Delete(prName, &metav1.DeleteOptions{}); err != nil {
-			err = fmt.Errorf("failed to delete pipelinerun %q: %s", prName, err)
-			errs = append(errs, err)
-			fmt.Fprintf(s.Err, "%s\n", err)
-			continue
-		}
-		success = append(success, prName)
-	}
-
-	if len(success) > 0 {
-		fmt.Fprintf(s.Out, "PipelineRuns deleted: %s\n", names.QuotedList(success))
-	}
-
-	return multierr.Combine(errs...)
+	d := deleter.New("PipelineRun", func(pipelineRunName string) error {
```

Comment:
Not that this should happen in this pr, but it may be a good idea to have TaskRuns be a related resource for pipelineruns.